### PR TITLE
Insert vertex only

### PR DESCRIPTION
**Pull request by cangfengzhs**
#### What type of PR is this?
- [ ] bug
- [x] feature
- [ ] enhancement

#### What does this PR do?
add `insert vertex only` statement
For example:
```
Insert vertex (1,2,3);
```
About parentheses in statement.at the beginning, I wanted to implement it in the following format, but there was ambiguity:
```
Insert vertex t();  // stmt1
Insert vertex t() Values 1:(); // stmt2
```
In stmt1, `t()` represents a function, but in stmt2, `t()` represents a tag

#### Which issue(s)/PR(s) this PR relates to?
part of issue: https://github.com/vesoft-inc/nebula/issues/3123
need PR: https://github.com/vesoft-inc/nebula/pull/3328
  
#### Checklist：
- [ ] Documentation affected （Please add the label if documentation needs to be modified.)
- [ ] Incompatible （If it is incompatible, please describe it and add corresponding label.）
- [ ] Need to cherry-pick （If need to cherry-pick to some branches, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes：
Please confirm whether to reflect in release notes and how to describe:
>                                                                 `

### CODE REVIEW

**Comment by Shylock-Hg** [comment]
How about `insert vertex values 1:(), 2:() ...`?

**Comment by cangfengzhs** [comment]
> How about `insert vertex values 1:(), 2:() ...`?

This problem has been initially solved. `insert vertex ifNotExists? VALUES vid_list` syntax is used.

File: src/graph/validator/MutateValidator.h [hunk 1]
```diff
@@ -38,6 +38,20 @@ class InsertVerticesValidator final : public Validator {
   std::vector<storage::cpp2::NewVertex> vertices_;
 };
 
+class InsertVerticesOnlyValidator final : public Validator {
```
Comment (Shylock-Hg) [inline src/graph/validator/MutateValidator.h:41]: Why don't reuse InsertVerticesValidator?

File: src/parser/Sentence.h [hunk 2]
```diff
@@ -52,6 +52,7 @@ class Sentence {
     kDropTag,
     kDropEdge,
     kInsertVertices,
+    kInsertVerticesOnly,
```
Comment (Shylock-Hg) [inline src/parser/Sentence.h:55]: ditto

File: src/graph/validator/MutateValidator.h (hunk 1 above)
Comment (cangfengzhs) [reply src/graph/validator/MutateValidator.h:41]: The syntax of `InsertVerticesOnly` and the syntax of `InsertVertices` are quite different in the parser process. This causes the data structure contained in the two Sentences to be different. If both Sentences use `InsertVerticesValidator`, two completely different `toPlan` logics are also required. So simply use a new `InsertVerticesOnlyValidator`. This also ensures that the impact on `InsertVertices` is minimal, and other bugs will not be introduced during the modification process.

Comment (Shylock-Hg) [reply src/graph/validator/MutateValidator.h:41]: https://github.com/vesoft-inc/nebula/blob/df3781cb1b530d93a6aaaff0f032cf626884f2a5/src/parser/parser.yy#L2839

I think just need add a new reduce rule like `vid { $$ = new VertexRowIten($1, new ValueList())}` in `vertex_row_item` rule.

Comment (cangfengzhs) [reply src/graph/validator/MutateValidator.h:41]: But, `InsertVertices` can not use vertexID list as "1,2,3".It needs "1:(),2:(),3:()" instead. So if add a new reduce rule in `vertex_row_item`, the parse process of `insertVertices` will be very strange.

File: tests/tck/features/insert/insertVertexOnly.feature [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      CREATE EDGE e();
+      """
+    And wait 6 seconds
+    When executing query:
+      """
+      INSERT VERTEX VALUES 1:(),2:(),3:();
+      INSERT EDGE e() VALUES 1->2:(),2->3:();
+      """
+    Then the execution should be successful
+    When executing query:
```
Comment (Shylock-Hg) [inline tests/tck/features/insert/insertVertexOnly.feature:25]: Add test case like `fetch prop on * 1`

**Review by wenhaocs — COMMENTED** [review]
In previous conversations, there were more files. Just want to make sure this PR contains the files you want. Otherwise, good to go after fixing shylock's comment.

**Comment by cangfengzhs** [comment]
> In previous conversations, there were more files. Just want to make sure this PR contains the files you want. Otherwise, good to go after fixing shylock's comment.

I adjust the implement according to shylock's suggestion and use another way which minimize the changes of previous code.

File: src/clients/storage/StorageClient.cpp [hunk 4]
```diff
@@ -218,11 +218,12 @@ StorageRpcRespFuture<cpp2::GetPropResponse> StorageClient::getProps(
     req.set_space_id(param.space);
     req.set_parts(std::move(c.second));
     req.set_dedup(dedup);
-    if (vertexProps != nullptr) {
```
Comment (Shylock-Hg) [inline src/clients/storage/StorageClient.cpp:221]: Why change this?

Comment (Shylock-Hg) [reply src/clients/storage/StorageClient.cpp:221]: wait fix in https://github.com/vesoft-inc/nebula/pull/3443

Comment (wenhaocs) [reply src/clients/storage/StorageClient.cpp:221]: The same. Fixing something else?

Comment (Shylock-Hg) [reply src/clients/storage/StorageClient.cpp:221]: @cangfengzhs Please check why vertexProp equals to null, maybe not caused by #3443

Comment (cangfengzhs) [reply src/clients/storage/StorageClient.cpp]: `FetchVerticesPlanner::buildVertexProps` return a `nullptr` when props is empty

**Review by Shylock-Hg — APPROVED** [review]

**Review by nevermore3 — APPROVED** [review]

**Pull request merged**